gems/smithy-client/lib/smithy-client/base.rb

```diff
@@ -9,9 +9,9 @@ class Base
       # @api private
       @plugins = PluginList.new(
         [
-          Plugins::Endpoint
+          Plugins::Endpoint,
```

Comment:
I assume you need this temporarily for testing until I actually merge the endpoints?

Reply:
It's part of your change is it not? I'm assuming it's still valid until you merge yours.